src/hooks/useSubmitForm.ts

```diff
@@ -0,0 +1,44 @@
+import axios from 'axios';
+import Cookies from 'js-cookie';
+import * as React from 'react';
+
+interface IField {
+  name: string;
+  value: string;
+}
+
+export const useSubmitForm = (
+  portalId: string,
+  formId: string,
+): [boolean, { success: string; error: string }, (fields: IField[]) => void] => {
+  const [loading, setLoading] = React.useState(false);
+  const [response, setResponse] = React.useState({ success: '', error: '' });
+
```

Comment:
Return null if theres no portal or form id in case someone forgets to add it

```suggestion
    if (! portalId || !formId) return null;
```

Reply:
Done